front/lib/api/data_sources.ts

```diff
@@ -170,3 +176,41 @@ async function warnPostDeletion(
       break;
   }
 }
+
+export async function enhanceDataSourceWithConnector(
```

Comment:
`nit` To align naming with other methods.
```suggestion
export async function augmentDataSourceWithConnector(
```